### ISSUE TITLE
SelfControl won't unblock

### ISSUE THREAD

**Issue by undertow47** [SelfControlApp/selfcontrol#532]
Hi there, 
I have a SelfControl question and have thoroughly read and worked through the FAQ. I can't get SelfControl to unblock (on Chrome on a MacBook Air). The timer ran down about 6-8 months ago (my blacklist is still available to me through Chrome's incognito feature).

I've tried SelfControl Killer, deleting the app, refreshing with the new version and everything from FAQ, but nothing works. I've searched online and through your wiki, but can't seem to find a solution to this. Would really appreciate your help. Thanks.

**Comment by cstigler** [comment]
Hi @undertow47, try:

1) Download/install the newest version of SelfControl from our website

2) if it shows the "start" screen, add one website and start a block for the shortest possible time

3) once that's done, or if it's already on the timer screen, wait for the block to expire

4) if it stays stuck on 00:00, click the "Stuck? Stop block manually" button below the timer, and use that to clear the block.

5) quit SelfControl

6) clear your browser cache

7) restart your computer

At that point, SelfControl's block will be cleared from your computer.

**Comment by undertow47** [comment]
Hi there,

Thanks for your suggestion. I've tried it, but still have the same issue.
The website I'd blocked using your suggested steps is unblocked, but not
the other websites on the original blacklist. Also (if it's relevant), when
I try to go to those original blocked sites, the page reverts to the Google
homepage (which is what I'd set it to do at the time), rather than the
standard Self Control default of 'the page cannot be found' etc, so it
seems to be an old block that's somehow stuck and looping.

I'd be grateful for any further help you can provide.

Thanks,
Karen.

**Comment by cstigler** [comment]
Hi Karen - SelfControl doesn't have the option to redirect blocked sites to the Google homepage. If SelfControl is blocking a page, you'll get the standard "cannot reach server" type messages.

It sounds like either you blocked those sites using software other than SelfControl, or you did some custom modification on the block after setting it up. Unfortunately I can't help with the networking issues beyond getting SelfControl's block off, which you've already done!

But I will give you one tip: if pages are redirecting to the Google homepage, it sounds like that's probably happening in your system's hosts file. I'd file these steps to restore your default hosts file: http://osxdaily.com/2014/04/12/restore-original-hosts-file-mac/
